developer-docs/bolt-api-servers.md

```diff
@@ -506,6 +507,208 @@ This returns a JSON object of this shape:
 }
 ```
 
+### GET /project_tasks
+- `project_ref`: String, *required* - Reference to the bolt project (in the form [PROJECT NAME]\_[SHA])
```

Comment:
nitpick, but we used `<PROJECT_NAME>_<REF>` rather than `<SHA>` for the form of the `project_ref` param in the orch API docs, should it match up here?

Reply:
Yeah i can update that.